.ebextensions/packages.config

```diff
@@ -0,0 +1,23 @@
+files:
+    "/opt/elasticbeanstalk/hooks/appdeploy/post/99_node.sh":
+        mode: "000755"
+        owner: root
+        group: root
+        content: |
+            #!/bin/bash
+
+            cd /var/app/current
+
+            #We probabaly should get this from a trusted source. For testing this is ok.
+            curl -o- curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+
+            # Source nvm
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+            # Install node
+            nvm install --lts
+
+            cd /var/app/current/web/themes/custom/move_mil/
+            npm install && npm run build
```

Comment:
there is a line feed missing. 
Also, I think Scott executes another command for building, something like npm run-script build, maybe is the same?

Reply:
we checked it out, looks like npm run build should be just fine even with the recent changes